components/explorer/SourceDetails/index.js

```diff
@@ -44,8 +44,8 @@ export default function SourceDetails({ data, setData, handleClose }) {
     distance,
     title,
     characteristics,
-    ra,
-    dec,
+    _RA: ra,
+    _DEC: dec,
```

Comment:
after updating to React 17 the "Zoom in" button on the object details stopped working and produced an undefined error. The component was trying to reference `ra` and `dec` which was not in the props, the props instead contained private versions called `_RA` and `_DEC`. Re-mapping these to the expected public prop values restored functionality.

Reply:
good catch